Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp

```diff
@@ -275,6 +276,15 @@ void ScrollingStateScrollingNode::setScrollerImpsFromScrollbars(Scrollbar*, Scro
 }
 #endif
 
+void ScrollingStateScrollingNode::setMouseIsInScrollbar(std::pair<bool, bool> flags)
```

Comment:
This std::pair may read better as a struct. It is very unclear that it represents right now.

Reply:
Changed to struct;